### DISCUSSION TITLE
How to Change the version of MicrosoftTeams.min.js

### DISCUSSION THREAD

**Discussion by Rodien** [OfficeDev/microsoft-365-agents-toolkit#9302] [General]
Hi,

I have a question about the teamsJs file that is being used. I am running Blazor Server and I would like to use another version of MicrosoftTeams.min.js. How can I make this change?

I am using Blazor Server.

![image](https://github.com/OfficeDev/TeamsFx/assets/50739829/cb96b265-95c2-4c86-9a4c-e774bfcd9cd4)

**Comment by therealjohn** [comment]
@Rodien In your project, there should be an `/Interop/InteropModuleBase.cs` file with this code:

```csharp
if (_module == null)
{
    _ = await ImportPrerequisiteModuleAsync("https://unpkg.com/@microsoft/teams-js@2.7.1/dist/MicrosoftTeams.min.js");
    _module = await _jsRuntime.InvokeAsync<IJSObjectReference>("import", ModulePath).AsTask();
}
```

You can modify that. Just note that if change versions and something in the MicrosoftTeams JS library has changed, the deserealization of the responses may not work because the classes in the `/Interop/TeamsSDK` folder are different. You'd need to update them to match the updated JS library too.

**Comment by Rodien** [comment]
Ah, okay. Thank you!

I am facing a problem with a blazoer server application (Teams app) where the user teams-js@2.4.* and taskmodule, as well as the graph toolkit components, are not working. 

However, they do work in the app that uses 2.7.1. 

So, I want to change 2.4.* to 2.7.1.
